### ISSUE TITLE
Icon Request: Techie (Brand)

### ISSUE THREAD

**Issue by mudit00** [FortAwesome/Font-Awesome#18859]
<svg width="360" height="157" viewBox="0 0 360 157" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M0 70.6837C6.81207 71.0846 12.0213 69.0809 19.6351 67.0772C22.8407 66.2758 25.2453 65.4745 28.4509 64.6731L46.8841 59.8643C50.0897 59.0629 54.4976 58.662 57.3026 57.8606C62.5123 55.8569 70.5266 55.0556 76.9381 53.4528C80.9451 52.6514 94.9705 51.0485 99.3784 51.0485C110.198 51.449 113.403 51.85 109.797 58.6621C108.194 61.0666 106.992 61.868 105.389 64.2722C92.1654 81.904 82.548 94.3262 72.5303 110.355C68.1221 117.568 63.3137 125.182 60.9091 133.196C58.9058 140.809 58.5049 148.023 63.7142 154.434C65.3173 155.636 68.5229 157.64 71.328 155.636C71.328 155.636 83.7502 130.391 87.7575 124.38C92.9668 116.366 96.5733 110.355 103.385 101.539C106.19 97.5318 108.995 94.3263 111.801 90.7198L125.025 75.0916C129.032 69.8823 139.851 58.2615 141.855 53.0519C143.457 49.0449 145.06 43.4347 140.252 39.8282C138.248 38.626 134.241 37.4241 130.634 37.0232C127.829 37.0232 111.801 39.0269 108.595 39.4278C99.7789 41.0306 88.9595 43.0342 78.9415 45.8389L7.61346 66.6764C4.80838 67.8787 1.20193 68.68 0 70.6837Z" fill="black"/>
<path d="M148.667 144.416C147.865 144.015 148.266 144.015 146.663 144.015C143.057 144.416 131.035 149.225 119.014 138.005C112.202 132.795 111.801 119.571 128.631 124.781C134.241 125.983 133.039 129.589 127.429 133.196C126.627 133.596 119.815 136.802 125.025 136.402L127.028 135.6C133.84 132.795 156.281 125.983 146.663 119.171C134.642 110.355 99.779 124.38 101.382 139.608C102.584 144.416 104.588 147.221 108.194 149.225C124.624 160.044 135.443 152.831 146.663 146.42C148.266 145.218 149.068 145.618 148.667 144.416Z" fill="black"/>
<path d="M198.757 146.419C195.952 144.816 198.757 144.816 194.75 145.217C191.945 145.618 190.743 145.618 188.338 145.618C181.927 145.217 176.717 142.412 173.111 139.607C157.883 126.383 176.317 117.167 185.934 127.185C186.735 127.986 187.136 129.188 187.938 129.989C190.743 129.188 200.36 125.582 200.761 122.777C201.562 118.369 186.335 112.358 168.703 121.174C158.685 126.383 150.67 132.795 155.88 142.011C157.082 144.816 162.692 149.625 165.096 151.228C172.71 156.036 184.732 153.231 191.945 150.025L196.753 147.621C198.757 146.82 197.555 148.022 198.757 146.419Z" fill="black"/>
<path d="M200.359 104.345C201.161 104.345 200.359 104.345 201.561 104.345C203.164 104.345 205.568 103.142 207.572 103.944C213.583 105.947 211.98 142.814 211.178 151.229C211.178 153.232 210.778 152.832 213.182 154.034C214.384 153.232 213.583 154.034 214.385 152.832L218.391 148.023C220.395 145.218 221.998 144.016 224.402 142.012C228.009 139.207 241.233 127.586 248.446 129.99C254.056 132.395 255.659 150.026 255.659 154.034C258.864 152.431 268.081 148.424 270.485 148.023C270.485 142.012 268.081 128.388 265.276 122.377C264.474 120.373 263.673 117.168 259.666 117.168C250.85 116.366 232.817 131.994 227.608 134.799C227.208 129.189 227.608 123.98 227.608 118.77C227.608 113.16 227.208 107.55 227.208 102.341C226.807 90.7201 219.995 95.128 202.764 101.94C201.161 102.742 199.959 103.142 200.359 104.345Z" fill="black"/>
<path d="M274.493 124.78C277.298 126.383 277.298 124.78 282.106 125.181C286.113 127.185 285.312 133.997 285.713 138.004C286.113 143.213 285.713 148.423 286.114 153.232C290.922 151.629 297.334 148.022 302.143 146.82C302.543 139.206 302.143 126.784 300.94 119.571C300.54 117.567 299.337 115.564 295.33 116.365C292.926 116.365 283.309 121.174 277.298 122.777L275.695 123.979C274.493 124.38 274.894 123.979 274.493 124.78Z" fill="black"/>
<path d="M275.295 107.549C277.298 110.354 281.306 113.56 288.118 111.957C289.721 111.557 299.338 106.748 300.139 106.347L297.735 105.145C293.728 103.943 293.728 102.741 290.121 100.737C286.114 102.34 277.699 105.946 275.295 107.549Z" fill="#F5821F"/>
<path d="M359.845 145.217L358.643 144.014C343.416 147.22 336.203 144.014 329.391 138.004C322.178 131.993 324.181 119.17 339.809 124.78C345.42 126.383 343.015 130.39 338.207 133.195C337.405 133.596 330.593 136.801 335.802 136.401L337.806 135.599C346.622 132.394 365.456 125.982 358.243 119.571C351.831 113.96 337.405 116.765 327.788 121.173C315.366 127.184 304.947 139.206 318.572 148.422C321.376 149.625 326.185 152.83 331.394 154.032C337.806 155.235 343.817 153.231 347.824 151.628L359.845 145.217Z" fill="black"/>
<path d="M2.40356 63.4704C6.41091 63.4704 8.41427 61.0662 14.0244 58.6616C51.6923 39.4273 80.5441 30.6112 125.825 19.7918C145.461 14.9834 167.099 13.3802 187.536 10.5751C195.15 9.77372 219.995 8.57179 221.998 7.77039C221.998 5.76672 219.594 5.76675 217.19 4.96535C215.186 4.16395 212.782 2.96171 210.778 2.16031C183.128 -8.25858 82.9482 21.7955 58.9052 33.0158C48.8872 37.0228 4.80776 58.2611 2.40356 63.4704Z" fill="#F5821F"/>
<path d="M48.0859 48.243C50.8907 49.4449 71.3276 43.0338 75.3346 42.232C97.3743 36.2214 117.811 31.8135 142.656 28.207L169.103 24.6005C172.71 24.1997 175.515 24.1996 179.121 23.7991C178.72 22.1963 176.316 21.3946 173.511 20.5932C169.103 19.3912 168.302 19.7918 163.894 20.1927C130.233 23.3983 84.5513 34.2177 54.8981 45.438C52.8944 46.2394 48.8873 46.6402 48.0859 48.243Z" fill="#979797"/>
</svg>
![Techie Logo](https://user-images.githubusercontent.com/43563263/160250676-04d98ef1-d6ab-4976-825c-07beeae75bad.png)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Could you please provide more details about this brand? An official website and how this brand can be used by other Font Awesome users would help

**Comment by aethior** [comment]
According to the attached logo, thi is an Indian webshop: https://techieindia.in/

**Comment by mudit00** [comment]
Yes! The Company domain is techieindia.in and its online store is at
techiestore.in.

**Comment by mudit00** [comment]
Sure! This brand provide laptop, mobile, computer  and printer accessories.
There store website is techiestore.in . I believe multiple Indian companies
and users of this company will use to showcase if there products are used
in there devices.
I am a freelancer so while making the websites or web app of clients many
time i came to the projects  where i have to use it.

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
